app/src/androidTest/java/com/sdp/movemeet/view/navigation/ActivityDescriptionNavigationTest.java

```diff
@@ -15,13 +19,12 @@
 
 public class ActivityDescriptionNavigationTest {
     @Rule
-    public ActivityScenarioRule<HomeScreenActivity> testRule = new ActivityScenarioRule<>(HomeScreenActivity.class);
+    public ActivityScenarioRule<LoginActivity> testRule = new ActivityScenarioRule<>(LoginActivity.class);
```

Comment:
It's not stupid, you're right to do that, it takes a step away!

Reply:
thanks haha